source/game.screen.supermarket.production.bmx

```diff
@@ -1666,9 +1668,34 @@ Type TGUISelectCastWindow Extends TGUIProductionModalWindow
 		castSelectList.filteredGenderID = filterToGenderID
 		'initial sorting
 		castSelectList.entries.sort(True, TGUICastSelectList.SortCastByName)
+
+		'instead of yes/no, this could return the age...
+		'TODO this filter does not prevent moving an underage actor to a director slot
+		Function filterAge:Int()
+			Local handler:TScreenHandler_SupermarketProduction = TScreenHandler_SupermarketProduction.GetInstance()
+			Local script:TScriptBase = handler.currentProductionConcept.script
+			Local jobSlotList:TGUICastSlotList = handler.castSlotList
+			Local jobId:Int = jobSlotList.selectCastWindow.selectJobID
+
+			If jobId = TVTPersonJob.DIRECTOR Then Return True
+			If jobId = TVTPersonJob.HOST Then Return True
+			If jobId = TVTPersonJob.REPORTER Then Return True
+			If script
+				If script.IsXRated() Then Return True
```

Comment:
Ich denke in einem FSK18-Drama kann es auch Jugendliche geben.

Eventuell wirklich etwas spezifischer vorgehen:
- FSK18? Dann sollten "Host", "Reporter" und "Regisseur" volljaehrig sein (Darsteller bzw "Gäste" oder "Sänger" hingegen koennten minderjaehrig sein)
- Erotik? Nur volljaehrige Personen erlauben (der Einfachheit halber)

generell kann man vlt definieren, dass ein "Host" oder "Reporter" (sowie Regisseur) ein Mindestalter haben sollte. der "rasende Reporter" einer Kindersendung koennte ja ab 12 gehen.